crates/typst-library/src/text/quote.rs

```diff
@@ -59,11 +63,12 @@ pub struct QuoteElem {
     ///
     /// ```example
     /// #set text(lang: "de")
+    /// Ein deutsch-sprechender Author zitiert unter umständen JFK:
     /// #quote[Ich bin ein Berliner.]
     ///
     /// #set text(lang: "en")
-    /// #set quote(quotes: true)
-    /// #quote(block: true)[I am a Berliner.]
+    /// And an english speaking one may translate the quote:
+    /// #quote[I am a Berliner.]
```

Comment:
could you wrap these at 37 cols so that they fit side-by-side in the docs?

Reply:
Should this generally be the case in doc comments? And is this the total or starting from the comment token?

Reply:
Starting after `/// ` . Not generally for doc comments, only for Typst code examples with the `preview` tag. It's not a hard and fast rule (sometimes, it reduces legibility), but I at least always try whether there's a good way to fit the example into 37 cols because it just looks better side-by-side in the docs.

Reply:
Well, I tried, the examples are a bit hard to put into 37 cols without making them unnecessarily long in height, so I went for a "best effort" approach. The examples in question are all `example` and not `preview`, so it should be fine. They should definitely look better than they did before.

Reply:
Sorry, I meant `example`, not `preview`.